packages/app/components/claim/claim-form.tsx

```diff
@@ -223,17 +227,43 @@ export const ClaimForm = ({ edition }: { edition: CreatorEditionResponse }) => {
             </Text>
           </View>
           <View tw="mt-4">
-            <Button
-              disabled={state.status === "loading"}
-              tw={state.status === "loading" ? "opacity-45" : ""}
-              onPress={handleClaimNFT}
-            >
-              {state.status === "loading"
-                ? "Claiming..."
-                : state.status === "error"
-                ? "Failed. Retry!"
-                : "Claim for free"}
-            </Button>
+            {shouldShowSignMessage ? (
+              <View tw="px-2">
+                {!isSignRequested ? (
```

Comment:
i think claim works without manual signature request, so we can prevent this prompt here

Reply:
during drop/mint, it uploads and takes a bit longer so it would need the signature prompt button, but claim should work!